.github/workflows/main.yml

```diff
@@ -58,9 +58,12 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: tarpaulin
-          args: --verbose --count --workspace --timeout 180 --out Html
+          args: --verbose --count --workspace --timeout 300 --out Xml
```

Comment:
Just curious -- what's the motivation for switching to `Xml`?

Reply:
So they could be input to CodeCov which makes the nice inline comment on the PR/has a web UI. I tried to output both xml and html but I think I needed a comment between args.

Reply:
Cool! I see the graphic now, it looks great